pkgs/applications/networking/owncloud-client/libre-graph-api-cpp-qt-client.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchurl, mkDerivation, cmake, qtbase }:
+
+mkDerivation rec {
+  pname = "libre-graph-api-cpp-qt-client";
+  version = "0.13.2";
+
+  srcs = (fetchurl {
+    url = "https://github.com/owncloud/${pname}/archive/refs/tags/v${version}.tar.gz";
+    sha256 = "sha256-ON0gpGaNz9honvyqppmj3C1jiHh99MciB5kgEwGXIAU=";
+  });
```

Comment:
Can you change this to `fetchFromGitHub`?

Reply:
done